text/slugify.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+
+/**
+ * **UNSTABLE**: New API, yet to be vetted.
```

Comment:
For future reference, this being the first paragraph means it's used as the summary for this symbol. Please ensure the description is always first for symbols.

Reply:
This is the same as what CLI does. https://docs.deno.com/api/deno/all_symbols

If you suggest something different, I think you should also suggest the same to CLI docs.

Reply:
https://github.com/denoland/deno/issues/25103